suites/quincy/cephadm/tier-0_5-1_cephadm.yaml

```diff
@@ -0,0 +1,819 @@
+#===============================================================================================
```

Comment:
Is this same testsuite as `cephadm/tier-0_cephadm.yaml` ?

Reply:
cephadm/tier-0_cephadm.yaml - this suite has most of the test cases from tier-0_5-1_cephadm.yaml. Added the missing tests from tier-0_5-1_cephadm.yaml to tier-0_cephadm.yaml suite and remove the duplicated suite.